docs/src/css/custom.css

```
@@ -352,108 +560,326 @@ html[data-page='home'][data-theme='dark'] .dropdown>.navbar__link:after {
+/* Widen the right-side TOC panel and pull it inward */
+.table-of-contents,
+[class*='tocCollapsible'],
+div[class*='tableOfContents'] {
+  min-width: 220px;
+  padding-left: 1rem;
+}
+
+div[class*='col--3'],
+div[class*='col col--3'] {
+  --ifm-col-width: calc(3 / 12 * 100%);
+  padding-left: 0 !important;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)